### PR TITLE
Blog/PagerTabStrip

### PR DESCRIPTION
**Pull request by mtnbarreto**

### CODE REVIEW

File: _posts/2021-07-30-Introducing-PagerTabStrip-SwiftUI-pager-view.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+What if I told you that with the PagerTabStrip you can create those pagers in a matter of minutes and not days. This exactly was our vision when we started to build this flexible yet powerful library.
+
+Let's move on and see how you can build a sophisticated pager view in terms of seconds, a code snippet worth more than a thousand words 😂.
+
+
+### Why we built PagerTabStrip!
+
+Nowadays almost every popular iOS app contains a pager view, the benefits are clear, often the app information architecture works best with a pager since it's simple to switch among these "equally relevant" information. Apps like Youtube, Instagram, Spotify, and Twitter use pager views to bring better UX but none of these pagers can be created using the native TabView. Sad, really?
+
```
Comment (milezabaleta) [inline _posts/2021-07-30-Introducing-PagerTabStrip-SwiftUI-pager-view.md:28]: YouTube goes with uppercased T

File: _posts/2021-07-30-Introducing-PagerTabStrip-SwiftUI-pager-view.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+On top of that, we are the creators of XLPagerTabStrip, the most popular pager view in UIKit with almost 7k stars in Github. We also noticed the apple community is constantly asking for a powerful SwiftUI pager view that leverages SwiftUI capabilities such as declarative syntax and the state-driven way to update the UI. Let's be honest, it's hard to find reusable SwiftUI components apart from these developed by Apple, so we embrace the challenge to raise the state of community components and PagerTabStrip is our first step.
+
+### First step with PagerTabStrip
+
+From the beginning, we designed the library to provide the same development experience as using a native Apple component such as TabView. So if you're familiar with TabView, PagerTabStrip will seem super straightforward to use, you just need to add the child view and provide the tabBarInfo view for each one.
+
+This post is just intended to provide a brief introduction about the library so if you're interested in how to use PagerTabStrip I would suggest you take a look at its [github repository readme], you can also run the example project that is contained in PagerTabStrip workspace.
+
```
Comment (milezabaleta) [inline _posts/2021-07-30-Introducing-PagerTabStrip-SwiftUI-pager-view.md:36]: I think that is ... I would suggest you TO take a look at...

**Review by milezabaleta — APPROVED** [review]

File: _posts/2021-07-30-Introducing-PagerTabStrip-SwiftUI-pager-view.md (hunk 1 above)
Comment (mats-claassen) [reply _posts/2021-07-30-Introducing-PagerTabStrip-SwiftUI-pager-view.md:28]: Is `Sad, really?` really a question?

File: _posts/2021-07-30-Introducing-PagerTabStrip-SwiftUI-pager-view.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+Let's move on and see how you can build a sophisticated pager view in terms of seconds, a code snippet worth more than a thousand words 😂.
+
+
+### Why we built PagerTabStrip!
+
+Nowadays almost every popular iOS app contains a pager view, the benefits are clear, often the app information architecture works best with a pager since it's simple to switch among these "equally relevant" information. Apps like Youtube, Instagram, Spotify, and Twitter use pager views to bring better UX but none of these pagers can be created using the native TabView. Sad, really?
+
+On top of that, we are the creators of XLPagerTabStrip, the most popular pager view in UIKit with almost 7k stars in Github. We also noticed the apple community is constantly asking for a powerful SwiftUI pager view that leverages SwiftUI capabilities such as declarative syntax and the state-driven way to update the UI. Let's be honest, it's hard to find reusable SwiftUI components apart from these developed by Apple, so we embrace the challenge to raise the state of community components and PagerTabStrip is our first step.
```
Comment (mats-claassen) [inline _posts/2021-07-30-Introducing-PagerTabStrip-SwiftUI-pager-view.md:29]:
```suggestion
On top of that, we are the creators of XLPagerTabStrip, the most popular pager view in UIKit with almost 7k stars in Github. We also noticed the Apple community is constantly asking for a powerful SwiftUI pager view that leverages SwiftUI capabilities such as declarative syntax and the state-driven way to update the UI. Let's be honest, it's hard to find reusable SwiftUI components apart from these developed by Apple, so we embraced the challenge to raise the state of community components and PagerTabStrip is our first step.
```

File: _posts/2021-07-30-Introducing-PagerTabStrip-SwiftUI-pager-view.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+### First step with PagerTabStrip
+
+From the beginning, we designed the library to provide the same development experience as using a native Apple component such as TabView. So if you're familiar with TabView, PagerTabStrip will seem super straightforward to use, you just need to add the child view and provide the tabBarInfo view for each one.
+
+This post is just intended to provide a brief introduction about the library so if you're interested in how to use PagerTabStrip I would suggest you take a look at its [github repository readme], you can also run the example project that is contained in PagerTabStrip workspace.
```
Comment (mats-claassen) [inline _posts/2021-07-30-Introducing-PagerTabStrip-SwiftUI-pager-view.md:35]: `is contained in the PagerTabStrip xcworkspace.`, maybe?

Comment (mats-claassen) [reply _posts/2021-07-30-Introducing-PagerTabStrip-SwiftUI-pager-view.md:35]: GitHub

File: _posts/2021-07-30-Introducing-PagerTabStrip-SwiftUI-pager-view.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+### And we're planning much more...
+
+We believe that PagerTabStrip's first release is very capable and powerful but we plan to enhance it even further. We have plans to integrate every XLPagerTabStrip style among other functionalities. Check out the list of upcoming features in the github repository.
```
Comment (mats-claassen) [inline _posts/2021-07-30-Introducing-PagerTabStrip-SwiftUI-pager-view.md:39]:
```suggestion
We believe that PagerTabStrip's first release is very capable and powerful but we plan to enhance it even further. We have plans to integrate every XLPagerTabStrip style among other functionalities. Check out the list of upcoming features in the GitHub repository.
```

**Review by mats-claassen — COMMENTED** [review]
I think we should add a code snippet. i get that the idea is to not be too technical here but one code snippet is not too technical and 95% of people reading this most certainly are iOS programmers. Also, ` a code snippet worth more than a thousand words` 😂

File: _posts/2021-07-30-Introducing-PagerTabStrip-SwiftUI-pager-view.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+permalink: /blog/:title/
+---
+
+We're incredibly thrilled to announce PagerTabStrip, the most powerful pager view fully written in SwiftUI.
+
+Creating a production-ready pager view could take several hours, even days of an engineering team. Such is the demand for pager views that Apple introduced a pager version on top of its native TabView recently but it lacks flexibility, its customization level is so limited that ends up not working in most cases.
+
+What if I told you that with the PagerTabStrip you can create those pagers in a matter of minutes and not days. This exactly was our vision when we started to build this flexible yet powerful library.
+
+Let's move on and see how you can build a sophisticated pager view in terms of seconds, a code snippet worth more than a thousand words 😂.
```
Comment (mats-claassen) [inline _posts/2021-07-30-Introducing-PagerTabStrip-SwiftUI-pager-view.md:22]:
```suggestion
Let's move on and see how you can build a sophisticated pager view in terms of seconds, a code snippet is worth more than a thousand words 😂.
```

File: _posts/2021-07-30-Introducing-PagerTabStrip-SwiftUI-pager-view.md (hunk 1 above)
Comment (mtnbarreto) [reply _posts/2021-07-30-Introducing-PagerTabStrip-SwiftUI-pager-view.md:58]: Updated

File: _posts/2021-07-30-Introducing-PagerTabStrip-SwiftUI-pager-view.md (hunk 3 above)
Comment (mtnbarreto) [reply _posts/2021-07-30-Introducing-PagerTabStrip-SwiftUI-pager-view.md]: pushing the change

**Pull request closed without merging**